docs/api/context-bridge.md

```diff
@@ -61,6 +61,19 @@ The `contextBridge` module has the following methods:
 * `apiKey` string - The key to inject the API onto `window` with.  The API will be accessible on `window[apiKey]`.
 * `api` any - Your API, more information on what this API can be and how it works is available below.
 
+### `contextBridge.evaluateInMainWorld(evaluationScript)` _Experimental_
+
+<!-- TODO(samuelmaddock): add generics to map the `args` types to the `func` params  -->
+
+* `evaluationScript` Object
+  * `func` (...args: any[]) => any - A JavaScript function to evaluate. This function will be serialized which means
+      that any bound parameters and execution context will be lost.
+  * `args` any[] (optional) - The arguments to pass to the provided function. These arguments must
```

Comment:
Since there was some confusion in the review today about what valid `args` values are:
```suggestion
  * `args` any[] (optional) - An array of arguments to pass to the provided function. These arguments must
```

Reply:
Included in [f103624](https://github.com/electron/electron/pull/44411/commits/f1036247fadcc283109a4ebc56009ca1ed39e4e1). Couldn't commit the suggestion since I rebased locally